agent/agent-tooling/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     def includeProfiler = project.hasProperty("includeProfiler") ? project.includeProfiler.toBoolean() : false
     if (includeProfiler) {
         implementation project(":agent:agent-profiler:agent-service-profiler")
+        implementation project(":agent:agent-gc-monitor:gc-monitor-core")
```

Comment:
this seems useful outside of the profiler(?)

Reply:
It would be useful outside the profiler, as you can see if the reporting level is set we stream back telemetry events that should give a high level of detail about garbage collection activity compared to our current reporting, although below what a detailed gc log file or detailed JFR profile can provide. I put it behind this guard mostly as I did see this as beta functionality that to some extent is going hand in hand with the profile triggering. But given it should not increase the binary size we could pull it out so that it is included in the binary, but disabled by default until we feel it can come out of beta